roles/openshift_logging_curator/tasks/main.yaml

```diff
@@ -90,7 +90,7 @@
     es_host: "{{ openshift_logging_curator_es_host }}"
     es_port: "{{ openshift_logging_curator_es_port }}"
     curator_cpu_limit: "{{ openshift_logging_curator_cpu_limit }}"
-    curator_cpu_request: "{{ openshift_logging_curator_cpu_request }}"
+    curator_cpu_request: "{{ openshift_logging_curator_cpu_request | min_cpu(openshift_logging_curator_cpu_limit | default(none)) }}"
```

Comment:
do we need `| default(none)` for these? is there a reason `min_cpu` wouldn't return a value? also we didn't have a default before... 

Won't block on it, but it does seem unnecessary